home-screen-widget/profiles/components/Widget.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div>
+        <div class="widget__header">
+            <!-- Put here the title of your widget -->
+            Profiles
+        </div>
+        <ProfilesGrid :profiles="profiles" />
+    </div>
+</template>
+
+<script>
+import { createNamespacedHelpers } from 'vuex';
+import BeekeeperHomeScreen, { EventType } from '@beekeeper/home-screen-sdk';
+
+import ProfilesGrid from '~/profiles/components/ProfilesGrid.vue';
+import store from '~/profiles/store';
+
+/**
+ * Step 1: Define widget id
+ *
+ *
+ * This is the vue component of your widget that will be loaded onto the home screen.
+ * In order for the home screen to be aware of  the widget we need to register it. That's why we defined here
+ * a unique widget string id that will identify our widget type.
+ * We called it 'profiles' since this is the profiles widget
+ */
+export const WIDGET_ID = 'profiles';
```

Comment:
Can you link the configuration service Docs that use this ID?

Reply:
yes